scripts/travis_doxygen.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e
+
+script_folder=`dirname $0`
+pip install --user unidiff
+
+doxygen --version &> /dev/null
+if [[ $? -ne 0 ]]
```

Comment:
This will never succeed because you've `set -e` at the top, the script will error out already if the previous command failed. Please test the value of the entire command rather than testing `$?`.